<launch>

  <!-- Stage Node-->
  <node pkg="stage" type="StageCommunicator.py" name="StageCommunicator"/>

  <param name="Stage_Update_dt" type="double" value="0.010"/>

  <include file="$(find plate_tf)/launch/plate_stage_transforms.launch" />

  <node pkg="stage_action_server" type="StageActionServer.py" name="StageActionServer"/>
</launch>
